tasks/install-FreeBSD.yml

```diff
@@ -13,6 +13,13 @@
   with_items:
     - tzinfo-data
 
+- name: Patch fluentd (issue \#30)
+  patch:
```

Comment:
this task fails when ruby != 2.3 or fluentd != 0.12.14. 
conditionals should be used to avoid the failure or it should be documented that the role requires ruby 2.3 and fluentd 0.12.14.
the latest gem should be used instead of freebsd package if there is no reason.